docs/reference/troubleshooting/common-issues/hotspotting.asciidoc

```diff
@@ -0,0 +1,266 @@
+hot spotting was occurring on these two nodes, and not necessarily from the same
+root cause. 
+
+[discrete]
+[[causes]]
+==== Causes
+
+Historically, clusters experience hot spotting mainly as an effect of hardware, 
+shard distributions, and/or task load. We'll review these sequentially in order 
+of their potentially impacting scope.
+
+[discrete]
+[[causes-hardware]]
+===== Hardware
+
+Here are some common improper hardware setups which may contribute to hot 
+spotting:
+
+* Resources are allocated non-uniformly. For example, if one hot node is 
+given half the CPU of its peers. {es} expects all nodes on a 
+<<data-tiers,data tier>> to share the same hardware profiles or 
+specifications.
+
+* Resources are consumed by another service on the host, including other 
+{es} nodes. Refer to our <<dedicated-host,dedicated host>> recommendation.
+
+* Resources experience different network or disk throughputs. For example, if one 
+node's I/O is lower than its peers. Refer to 
+<<tune-for-indexing-speed,Use faster hardware>> for more information.
+
+* A JVM that has been configured with a heap larger than 31GB. Refer to <<set-jvm-heap-size>> 
+for more information.
+
+* Problematic resources uniquely report <<setup-configuration-memory,memory swapping>>. 
+
+[discrete]
+[[causes-shards]]
+===== Shard distributions
+
+{es} indices are divided into one or more link:{wikipedia}/Shard_(database_architecture)[shards] 
+which can sometimes be poorly distributed. {es} accounts for this by <<modules-cluster,balancing shard counts>> 
+across data nodes. As link:{blog-ref}whats-new-elasticsearch-kibana-cloud-8-6-0[introduced in version 8.6], 
+{es} by default also enables <<modules-cluster,desired balancing>> to account for ingest load; 
+though there's always link:{es-repo}/issues/17213[room for improvement]. 
+A node may still experience hot spotting either due to write-heavy indices or by the 
+overall shards it's hosting.
+
+[discrete]
+[[causes-shards-nodes]]
+====== Node level
+
+You can check for shard balancing via <<cat-allocation,cat allocation>>, though as of version 
+8.6, <<modules-cluster,desired balancing>> may no longer fully expect to 
+balance shards. Kindly note, both methods may temporarily show problematic imbalance during 
+<<cluster-fault-detection,cluster stability issues>>.
+
+For example, let's showcase two separate plausible issues using cat allocation:
+
+[source,console]
+----
+GET _cat/allocation?v&s=node&h=node,shards,disk.percent,disk.indices,disk.used
+----
+
+Which could return:
+
+[source,console-result]
+----
+node   shards disk.percent disk.indices disk.used
+node_1    446           19      154.8gb   173.1gb
+node_2     31           52       44.6gb   372.7gb
+node_3    445           43      271.5gb   289.4gb
+----
+// TEST[skip:illustrative response only]
+
+Here we see two significantly unique situations. `node_1` has 
+recently restarted, so it has a much lower number of shards than all other nodes. 
```

Comment:
@abdonpijpelink oops, this was suppose to reference `node_2`. I don't know how to edit my PR, can you help me fix it?

Support feedback would also like to add in this ballpark (about `node_2`'s situation) something to the effect of
```
While `node_2`'s shard count is low, it may become a index-write 
hot spot due with ongoing <<ilm-rollover,ILM Rollovers>>. This is a 
common root cause of write hot spots covered in the next section.
```

Reply:
That's my mistake. I'll fix it and add that note.